### DISCUSSION TITLE
Now Playing design ideas

### DISCUSSION THREAD

**Discussion by BlackSparkz** [zyrouge/symphony#342] [Ideas]
What about thiss button ui
![20231120_170452.jpg](https://github.com/zyrouge/symphony/assets/78287172/3cc56fa8-5587-46bf-a6c5-eac8516476ab)

![20231120_145254.png](https://github.com/zyrouge/symphony/assets/78287172/1950a805-b59b-45e6-8f61-1ec53b948fe1)

**Comment by zyrouge** [comment]
That doesn't look visually pleasing. I'll be converting this to a discussion, any finalized one might be implemented.

**Comment by zyrouge** [comment]
Closing this as stale.
